samples/tab-stage-view/csharp/README.md

```diff
@@ -45,7 +45,7 @@ For reference please check [Tabs link unfurling and Stage View](https://docs.mic
 3. Modify the `manifest.json` in the `/AppPackage` folder and replace the following details
    - `<<MANIFEST-ID>>` with some unique GUID   
```

Comment:
This should be <<GUID>>
Because in your manifest, it is like below:
"id": "<<GUID>>",